posthog/urls.py

```diff
@@ -240,6 +241,9 @@ def opt_slash_path(route: str, view: Callable, name: Optional[str] = None) -> UR
     path("", include("social_django.urls", namespace="social")),
     path("uploaded_media/<str:image_uuid>", uploaded_media.download),
     opt_slash_path("slack/interactivity-callback", slack_interactivity_callback),
+    # Message preferences
+    path("preferences/<str:token>/", preferences_page, name="message_preferences"),
+    path("preferences/update/", update_preferences, name="message_preferences_update"),
```

Comment:
preferences feels a little too generic.

I also wonder if we anyways would want to have this hosted slightly differently 🤔 I'm not sure how people will feel about landing on `us.posthog.com` (especially with adblockers etc.)